.github/workflows/cypress.yml

```
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  workflow_call:
+    inputs:
+      cypress_host:
+        description: host name
+        type: string
+        required: true
+    secrets:
+      CODECOV_TOKEN:
+        description: Code coverrage token
+        required: true
+      SLACK_WEBHOOK:
+        description: Wehook slack
+        required: true
+jobs:
+  concurrency:
+    group: "cypress-${{ github.workflow }}-${{ github.ref }}"
+  needs: ["deploy"]
+  runs-on: ubuntu-latest
+  steps:
+    - name: Checkout project
+      uses: actions/checkout@v4
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 20
+
+    - uses: actions/cache@v3
+      with:
+        path: |
+          **/node_modules
+          .yarn/install-state.gz
+          .yarn/cache
+        key: yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: yarn-
+
+    - name: Install dependencies
+      run: yarn install
+
+    - name: Run cypress e2e tests
+      run: yarn e2e:headless --env ui=${{inputs.cypress_host}},server=${{inputs.cypress_host}}
+
+    - name: Notify failure on Slack
+      uses: ravsamhq/notify-slack-action@v2
+      if: ${{ inputs.environment }} == 'recette'
```

Je penses qu'il te faut `failure` car par default en failure ça skip les steps suivants
```suggestion
      if: ${{ failure() }} && ${{ inputs.environment }} == 'recette'
```

---

Du coup failure() va rattraper ce qui aurait dû être skippé ?